source/deployment/deployContracts.ts

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env node
+
+import * as fs from 'async-file';
+import * as path from 'path';
+import * as HttpProvider from 'ethjs-provider-http';
+import * as Eth from 'ethjs-query';
+import * as EthContract from 'ethjs-contract';
+import { ContractBlockchainData } from 'contract-deployment';
+import { SolidityContractCompiler } from "../libraries/CompileSolidity";
+import { ContractDeployer } from "../libraries/ContractDeployer";
+import { RpcClient } from "../libraries/RpcClient";
+
+const CONTRACT_INPUT_DIR_PATH = path.join(__dirname, "../../source/contracts");
+const CONTRACT_OUTPUT_DIR_PATH = path.join(__dirname, "../contracts");
+const COMPILED_CONTRACT_OUTPUT_FILE_NAME = "augurCore";
+const GAS_AMOUNT = 3000000;
+const DEFAULT_ETHEREUM_PORT = 8545;
+
+// Set these variables to use node settings other than TestRPC on port 8545
+const ETHEREUM_HOST = "";
+const ETHEREUM_PORT = "";
```

Comment:
No need to have file variables for these, you can just use `process.env.ETHEREUM_HOST` down on line 30/31 instead (and similar for port).

Reply:
So that logic should look like the following?

```
    const httpProviderport = (typeof process.env.ETHEREUM_PORT == 'undefined') ? DEFAULT_ETHEREUM_PORT : process.env.ETHEREUM_PORT;
    if (typeof process.env.ETHEREUM_HOST == 'undefined') {
        const rpcClient = new RpcClient();
        await rpcClient.listen(httpProviderport);
    }
```

And we would set these values in a .env file?  (Sorry, I haven't worked with Node much before.  I did a search on how to set Node environment variables, and it sounds like this is how we would set them?)

Reply:
Yeah, I believe that is correct.

You would just export them before running the script:
```
export ETHEREUM_HOST = 'localhost'
export ETHEREUM_PORT = '8545'
node output/deployment/deployContracts.js
```